### ISSUE TITLE
Gtk-Warning when trying to start from command line.

### ISSUE THREAD

**Issue by jrichocean** [howl-editor/howl#275]
Im getting a warning and am unable to start the editor on my computer.
```(howl:7776): Gtk-WARNING **: Theme parsing error: <data>:44:46: Using Pango syntax for the font: style property is deprecated; please use CSS syntax
Failed to dispatch 'signal activate: ../lib/ljglibs/gtk/css_provider.moon:12: <data>:20:8not a number (code: 1)'```
Im on MacOSX 10.11.6
gtk version: 3.22.2
C compiler: llvm 8.0.0

**Comment by kirbyfan64** [comment]
You're using Howl 0.4, right? [This was fixed in 0.4.1.](http://howl.io/blog/2016/10/14/howl-0-4-1-released.html)

**Comment by jrichocean** [comment]
that was it, i didn't realize i didn't have the up to date repo to build. thanks
